src/blop/integrations/ax/agent.py

```diff
@@ -241,7 +272,7 @@ def acquire(self, trials: dict[int, TParameterization]) -> Generator[Msg, str, d
         """
         plan_args = self._unpack_parameters(trials.values())
         uid = yield from list_scan(self.readables, *plan_args, md={"ax_trial_indices": list(trials.keys())})
-        results_df = self.db[uid].table(fill=True)
+        results_df = self.convert_to_dictonary(self.db[uid])
```

Comment:
This should be `self.data_access.get_data()`
```suggestion
        results = self.data_access.get_data(uid)
```